packages/blockchain-wallet-v4-frontend/src/scenes/Signup/components/SignupForm/index.tsx

```diff
@@ -66,9 +65,37 @@ const PasswordRequirementText = styled(Text)<{ isValid?: boolean }>`
 
 const validatePasswordConfirmation = validPasswordConfirmation('password')
 
+const getCountryElements = (countries: Array<CountryType>) => [
+  {
+    group: '',
+    items: countries.map((country: CountryType) => ({
+      text: country.name,
+      value: country
+    }))
+  }
+]
+
+const getStateElements = (states: Array<StateType>) => [
+  {
+    group: '',
+    items: states.map((state: StateType) => ({
+      text: state.name,
+      value: state
+    }))
+  }
+]
```

Comment:
are these always US states?  if so, maybe rename things to indicated that? `usStates`?

Reply:
they are, yeah we can rename but BE expect to get just state will do convert when sending it back to BE